core/src/main/java/com/nisovin/magicspells/listeners/CastListener.java

```diff
@@ -67,15 +69,29 @@ public void onPlayerInteract(PlayerInteractEvent event) {
 			return;
 		}
 
-		if (event.getAction() == Action.LEFT_CLICK_AIR || event.getAction() == Action.LEFT_CLICK_BLOCK) {
+		if ((event.getAction() == Action.LEFT_CLICK_AIR || event.getAction() == Action.LEFT_CLICK_BLOCK) && (!MagicSpells.areBowCycleButtonsReversed() || (event.hasItem() && event.getItem().getType() != Material.BOW && event.getItem().getType() != Material.CROSSBOW))) {
```

Comment:
By checking if the material is a crossbow, you cut off 1.13 support (crossbows were added in 1.14), use `material.name().contains`